src/commands/node/status.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { flags as flagParser } from '@oclif/command';
+import Listr from 'listr';
+import BaseCommand from '../../base';
+import { NETWORK } from '../../utils/constants';
+import { flags as commonFlags } from '../../utils/flags';
+import { describeApplication, Pm2Env } from '../../utils/node/pm2';
+
+interface Flags {
+	readonly name: string;
+}
+
+export default class StatusCommand extends BaseCommand {
+	static description = 'Lisk Core Instance Status';
```

Comment:
`Show status of an installed Lisk Core instance`?

Reply:
it will be more verbose, how about `Status of Lisk Core Instance`?

Reply:
i think it should be verbose for description =0